collections/infrastructure/roles/repositories/tasks/RedHat/main.yml

```diff
@@ -20,7 +20,7 @@
 - name: yum_repository <|> Setting repositories
   ansible.builtin.yum_repository:
     name: "{{ item.name | default(item) }}"
-    description: "{{ item.description | default(item) }} gen by Ansible"
+    description: "{{ item.name | default(item) }} gen by Ansible"
```

Comment:
Can you change that to:

`"{{ item.description | default(item.name, true) | default(item, true) }} gen by Ansible"`

?
Idea is to allow description, and fallback on name then on item if not set or None